litmus/PTX/Manual/Ticketlock-acq2rlx-1.litmus

```diff
@@ -13,6 +13,6 @@ x=0;
  LC01:                             | LC01:                            ;
  ld.weak r3, x                     | ld.weak r3, x                    ;
  st.weak x, 1                      | st.weak x, 2                     ;
- atom.release.gpu.plus r1, out, 1  | atom.release.gpu.plus r1, out, 1 ;
+ atom.release.gpu.plus r4, out, 1  | atom.release.gpu.plus r4, out, 1 ;
```

Comment:
What does this instruction do? Is this like a FADD and `r4` gets the old value? Cause I'm confused why a register gets assigned in a release operation.

Reply:
Yes. Atoms are RMW and this is an FADD.